vendor/github.com/evanphx/json-patch/patch.go

```diff
@@ -389,17 +389,13 @@ func (d *partialArray) add(key string, val *lazyNode) error {
 
 	cur := *d
 
-	if idx < 0 {
-		idx *= -1
```

Comment:
didn't this already do negative indexing?

Reply:
SIGH